Installation/cmake/modules/process_dependencies.cmake

```diff
@@ -8,13 +8,17 @@ if(NOT CGAL_PACKAGES_PREFIX)
 endif()
 
 #message("regexp: \\. ${CGAL_PACKAGES_PREFIX}/[^/]*/include/CGAL/.*h")
+#TO BE CHECKED
+get_filename_component(INSTALLATION "${CMAKE_BINARY_DIR}" DIRECTORY)
+get_filename_component(BUILD_DIR "${INSTALLATION}" NAME)
```

Comment:
Could you comment what has to be checked?

Reply:
I think I just forgot to remove the commentary. It has been checked already.

Reply:
Thanks for the answer. I have pushed 3457d2442b603aac2f643711655268b3ea566345 that removes that line.

Reply:
I overrid it with more clean-up